packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -577,8 +582,54 @@ class SingleActivator with Diagnosticable, MenuSerializableShortcut implements S
 ///  * [SingleActivator], an activator that represents a single key combined
 ///    with modifiers, such as `Ctrl+C`.
 class CharacterActivator with Diagnosticable, MenuSerializableShortcut implements ShortcutActivator {
-  /// Create a [CharacterActivator] from the triggering character.
-  const CharacterActivator(this.character);
+  /// Triggered when the key event yields the given character.
+  ///
+  /// The [control] and [meta] flags represent whether the respect modifier
+  /// keys should be held (true) or released (false). They default to false.
+  /// [CharacterActivator] can not check Shift keys or Alt keys yet, and will
+  /// accept whether they are pressed or not.
+  ///
+  /// By default, the activator is checked on all [RawKeyDownEvent] events for
+  /// the [character]. If `includeRepeats` is false, only the [character]
+  /// events with a false [RawKeyDownEvent.repeat] attribute will be
+  /// considered.
+  const CharacterActivator(this.character, {
+    this.control = false,
+    this.meta = false,
+    this.includeRepeats = true,
+  });
+
+  /// Whether either (or both) control keys should be held for the [character]
+  /// to activate the shortcut.
+  ///
+  /// It defaults to false, meaning all Control keys must be released when the
+  /// event is received in order to activate the shortcut. If it's true, then
+  /// either or both Control keys must be pressed.
+  ///
+  /// See also:
+  ///
+  ///  * [LogicalKeyboardKey.controlLeft], [LogicalKeyboardKey.controlRight].
+  final bool control;
+
+  /// Whether either (or both) meta keys should be held for the [character] to
+  /// activate the shortcut.
+  ///
+  /// It defaults to false, meaning all Meta keys must be released when the
+  /// event is received in order to activate the shortcut. If it's true, then
+  /// either or both Meta keys must be pressed.
+  ///
+  /// See also:
+  ///
+  ///  * [LogicalKeyboardKey.metaLeft], [LogicalKeyboardKey.metaRight].
+  final bool meta;
+
+  /// Whether this activator accepts repeat events of the [character].
+  ///
+  /// If [includeRepeats] is true, the activator is checked on all
+  /// [RawKeyDownEvent] events for the [character]. If `includeRepeats` is
+  /// false, only the [character] events with a false [RawKeyDownEvent.repeat]
+  /// attribute will be considered.
```

Comment:
Seems like you could say something like "only the first [character] events in the repeated sequence will be considered." It just reads a little too technical.